grails-app/views/dataUse/edit.gsp

```diff
@@ -259,7 +288,8 @@
                               name="controlSetOption"
                               id="controlSetOptionUnspecified"
                               <g:if test="${restriction.controlSetOption?.equals("Unspecified") && !create}">checked</g:if>
-                              value="Unspecified">Unspecified</label>
+                              value="Unspecified">Unspecified                <tr>
```

Comment:
nit: alignment. `<tr>` should, I think, be on the next line to align with the closing `</tr>`

Reply:
Actually there's no need for this tags.